src/main/java/com/cloudbees/jenkins/support/filter/InetAddressContentFilter.java

```diff
@@ -66,7 +66,11 @@ public static InetAddressContentFilter get() {
         Set<ContentMapping> found = new HashSet<>();
         Matcher m = IP_ADDRESS.matcher(input);
         while (m.find()) {
-            found.add(mappings.getMappingOrCreate(m.group(), InetAddressContentFilter::newMapping));
+            String ip = m.group();
+
+            if (!mappings.getStopWords().contains(ip)) {
```

Comment:
Why don't just compare with Jenkins.VERSION? It seems like _stop words_ has nothing to do with the _Inet address filter_

Reply:
Agreed. IP address "stop words" would only be specific version numbers. In this scenario, you could always check the current Jenkins version. In a more thorough solution, you'd make the stop words essentially a whitelist of IP addresses which has UX along with it.

Reply:
Yes, since it could probably be interesting to use stop words in a more general way to prevent another IP addresses to be filtered, I'll keep that mechanism in the filter.